index.html

```diff
@@ -72,6 +72,27 @@
           <a>standardized payment method identifier</a>.
         </li>
       </ul>
+      <section>
+        <h3>
+          Validity
+        </h3>
+        <p>
+          The steps to <dfn>validate a payment method identifier</dfn> with a
+          <var>pmi</var> string are given by the following algorithm. It
+          returns either true or false.
+        </p>
+        <ol>
+          <li>Let <var>url</var> be the result of running the <a>basic URL
+          parser</a> with <var>pmi</var>.
```

Comment:
basic URL parser link isn't working in https://rawgit.com/w3c/payment-method-id/97fcb88628ab8a34c25d89b1caf435abf12d1e47/index.html